### ISSUE TITLE
Xtend Batch Compiler Main waits for ThreadExecutor

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-xtend#358]
Xtend Batch Compiler Main waits for the ThreadExecutor created by AnnotationProcessor
I propose to exit the main with {{System.exit(0)}} or to offer a second main that does so.
Or to explicitely shut the executor down (question would be how to access it)

**Comment by cdietrich** [comment]
[Main-2018-02-09-shutdown.snapshot.zip](https://github.com/eclipse/xtext-xtend/files/1711020/Main-2018-02-09-shutdown.snapshot.zip)

**Comment by cdietrich** [comment]
can be reproduced by calling `org.eclipse.xtend.core.compiler.batch.Main` with
```
-d /data/eclipse-workspaces/ws-dev-cdietric-largedevices/demo/src-gen2 -cp /home/cdietric/.p2/pool/plugins/com.google.guava_15.0.0.v201403281430.jar:/home/cdietric/.p2/pool/plugins/org.eclipse.xtext.xbase.lib_2.13.0.v20171012-0742.jar:/home/cdietric/.p2/pool/plugins/org.eclipse.xtend.lib_2.13.0.v20171012-0742.jar:/home/cdietric/.p2/pool/plugins/org.eclipse.xtend.lib.macro_2.13.0.v20171012-0742.jar /data/eclipse-workspaces/ws-dev-cdietric-largedevices/demo/src
``` 

on
```
package demo

import org.eclipse.xtend.lib.annotations.Accessors

class Demo {
    
    @Accessors
    var String x
    
}
```

**Comment by cdietrich** [comment]
possible solutions:

(1) shut down executor service
(2) system.exit(0)

**Comment by cdietrich** [comment]
Fixed in 2.14
